TsRandomizer/Settings/GameSettingsLoader.cs

```diff
@@ -99,6 +111,53 @@ public static SettingCollection LoadSettingsFromSlotData(Dictionary<string, obje
 
 				settings.ShopFill.Value = enumValue;
 			}
+
+      if (slotData.TryGetValue("DamageRando", out var damageRando))
+			{
+				var value = ToInt(damageRando);
+        string enumValue;
+
+				switch (value)
+				{
+					case 1:
+ 						enumValue = "All Nerfs";
+						break;
+
+					case 2:
+						enumValue = "Mostly Nerfs";
+						break;
+
+					case 3:
+						enumValue = "Balanced";
+						break;
+            
+ 					case 4:
+						enumValue = "Mostly Buffs";
+						break;
+
+					case 5:
+						enumValue = "All Buffs";
+						break;
+
+					case 6:
+						enumValue = "Manual";
+						break;
+
+					default:
+						enumValue = "Off";
+						break;
+				}
+				settings.DamageRando.Value = enumValue;
+			}
+      
+      if (settings.DamageRando.Value != "Off"
```

Comment:
Should this not be settings.DamageRando.Value == "Manual"? as i think you only need to look at the rando overrides if its set to be manually configured

Reply:
Yeah, the != "Off" was from an older idea that ended up being worse than what I ended up with